ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -402,6 +421,31 @@ open class NavigationBar: UINavigationBar, TokenizedControlInternal {
         }
     }
 
+    private func updateGradient() {
+        if style != .gradient {
+            return
+        }
+
+        guard let gradient = gradient else {
+            assertionFailure("gradient cannot be nil when using the gradient style")
```

Comment:
I don't think we want an assertion failure here. This would fail if the user sets the style to gradient, then sets the gradient mask before the gradient. I think just returning should be fine.